repos/system_upgrade/el7toel8/actors/postgresqlcheck/libraries/postgresqlcheck.py

```diff
@@ -18,7 +18,7 @@
 )
 
 # Link URL for postgresql-server report
-report_server_inst_link_url = 'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/deploying_different_types_of_servers/index#migrating-to-a-rhel-8-version-of-postgresql_using-postgresql'  # noqa: E501; pylint: disable=line-too-long
+report_server_inst_link_url = 'https://red.ht/rhel-8-deploying-different-types-of-servers'
```

Comment:
Are we sure this one is really the correct replacement?

Reply:
@Rezney  good catch! fixed